src/dotnet-install.sh

```diff
@@ -713,44 +732,56 @@ download() {
     return 0
 }
 
+# Note, function writes to global variables $http_code and $download_error_msg
```

Comment:
nit: probably better wording could be: `updates/sets global variables $http_code and $download_error_msg`. `Note, fuction` seems obvious for me